src/gui/wizard/webview.cpp

```diff
@@ -145,7 +147,12 @@ QWebEnginePage * WebEnginePage::createWindow(QWebEnginePage::WebWindowType type)
     ExternalWebEnginePage *view = new ExternalWebEnginePage(this->profile());
     return view;
 }
-
+    
+bool WebEnginePage::certificateError(const QWebEngineCertificateError &certificateError) {
+    //handle webview certificate errors here 
+    return true;
```

Comment:
Ok so after looking it up. really fixing it is not so straight forward as the webengineview uses it own network stack. So no QNAM etc.

What we could do. Is pass in the original URL we try to connect to. As we have to accept that cert before continuing.

If the url matches the cert here. Then we continue. Else we should still fail.

How does that sound?